include/dem/pp_linear_force.h

```diff
@@ -68,8 +70,10 @@ class PPLinearForce : public PPContactForce<dim>
       &local_adjacent_particles,
     std::unordered_map<int,
                        std::unordered_map<int, pp_contact_info_struct<dim>>>
-      &           ghost_adjacent_particles,
-    const double &dt) override;
+      &                                      ghost_adjacent_particles,
+    const double &                           dt,
+    std::unordered_map<int, Tensor<1, dim>> &momentum,
```

Comment:
momentum is u * mass? Not sure it is worth it to store it...

Reply:
Similar to force, we calculate momentum in the particle-particle and particle-wall contact force calculations. Then in the integration, we integrate it to calculate omega. So we need this momentum.